src/index.html

```diff
@@ -4,10 +4,8 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link
-            href="https://fonts.googleapis.com/css?family=PT+Sans:400,400i,700,700i"
-            rel="stylesheet"
-        />
+        <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,400i,700,700i" rel="stylesheet" />
+        <link href="../public/katex.min.css" rel="stylesheet" />
```

Comment:
Will not be loaded in combination with the schulcloud-client css has to be imported in the index.js or Application/index.js